src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -5084,7 +5084,7 @@ internal enum ErrorCode
             ERR_ExpressionTreeContainsBadCoalesce = 845,
             ERR_ArrayInitializerExpected = 846,
             ERR_ArrayInitializerIncorrectLength = 847,
-            ERR_OverloadRefOutCtor = 851,
+            ERR_OverloadRefKindCtor = 851,
```

Comment:
This instance of `ErrorCode` should probably match the original `ErrorCode`. In other words, we should rename both instances of `ERR_OverloadRefOutCtor` or neither.